01_prepare_host.sh

```diff
@@ -6,6 +6,11 @@ source lib/logging.sh
 # shellcheck disable=SC1091
 source lib/common.sh
 
+if [[ "${EPHEMERAL_CLUSTER}" == "minikube" ]] && [[ $(id -u) == 0 ]]; then
```

Comment:
maybe the test could be independent of the EPHEMERAL_CLUSTER, so that we always make sure not to run as root. wtr, there should not be differences between the two setups IMO.